--- a/components/safe_browsing/content/common/safe_browsing.mojom
+++ b/components/safe_browsing/content/common/safe_browsing.mojom
@@ -133,7 +133,6 @@
   VISUAL_EXTRACTION_FAILED = 10,
 };
 
-[EnableIf=full_safe_browsing]
 // Interface for starting phishing classification. This is scoped to a
 // particular RenderFrame.
 interface PhishingDetector {
@@ -146,7 +145,6 @@
         mojo_base.mojom.ProtoWrapper? request);
 };
 
-[EnableIf=full_safe_browsing]
 // Interface for setting a phishing model. This is scoped to an entire
 // RenderProcess.
 interface PhishingModelSetter {
@@ -217,7 +215,6 @@
   kFailed,
 };
 
-[EnableIf=full_safe_browsing]
 // Interface for starting image embedding. This is scoped to a
 // particular RenderFrame.
 interface PhishingImageEmbedderDetector {
